alpakaConfig.cmake

```diff
@@ -389,25 +395,25 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                         ENDIF()
                     ELSEIF(CUDA_VERSION VERSION_EQUAL 9.2)
                         IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 8.0)
-                            MESSAGE(FATAL_ERROR "NVCC 9.2 does not support GCC 8+. Please use GCC 4.9, 5, 6 or 7!")
+                            MESSAGE(FATAL_ERROR "NVCC 9.2 in combination with GCC is not yet supported by alpaka!")
                         ENDIF()
                     ENDIF()
                 ELSEIF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
                     IF(CUDA_VERSION VERSION_EQUAL 8.0)
                         IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 4.0)
-                            MESSAGE(FATAL_ERROR "NVCC 8.0 does not support clang 4+. Please use clang 3.9!")
+                            MESSAGE(FATAL_ERROR "NVCC 8.0 does not support clang 4+. Please use NVCC 9.0!")
```

Comment:
shouldn't you advise NVCC 9.1 then? (see below)

Reply:
👍